src/hypervisor.c

```diff
@@ -43,6 +43,62 @@
 private gchar *sysconfdir = SYSCONFDIR;
 private gchar *defaultsdir = DEFAULTSDIR;
 
+static gchar *
+cc_oci_expand_net_cmdline(struct cc_oci_config *config) {
+	/* www.kernel.org/doc/Documentation/filesystems/nfs/nfsroot.txt
+        * ip=<client-ip>:<server-ip>:<gw-ip>:<netmask>:<hostname>:
+         * <device>:<autoconf>:<dns0-ip>:<dns1-ip>
+	 */
+
+	/* FIXME. Sending multiple ip= does not seem to work
+	 * Hence support one or the other for now
+	 * Explore dracut or systemd based network init
+	 */
+
+	if ( config->net.ip_address != NULL ) {
+		return ( g_strdup_printf("ip=%s:::%s:%s:%s:off::",
```

Comment:
The plan to get a deafult gateway is use the namespace interface as well ? I've created an issue about this #55